wordpress-mobile/WordPress-Android
# Add line chart to ValueWithChartViewHolder

Fixes #16494 

This PR adds a line chart for the current week (last 7 days + today) to Total Likes and Total Comments cards.

|Light|Dark|
|-|-|
|<img src="https://user-images.githubusercontent.com/2471769/167645742-07cfe9c4-5499-4267-80c9-d1f4aff70f2c.png" width=320>|<img src="https://user-images.githubusercontent.com/2471769/167645886-fd6241cd-64eb-4584-b1ee-12079a4c2908.png" width=320>|

To test: 
1. Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings).
2. Select Debug Settings.
3. Find StatsRevampV2FeatureConfig under Features in development enable it and restart app.
4. Go to stats either using quick links or menu.
5. Ensure that it opens in Insights tab otherwise switch to Insights tab.
6. If the Total Likes and/or Total comments cards are not present, go to the stats management (either through ⚙️ menu at the top right hand corner or Add new stats card at the bottom) and add cards and save, then return to Insights.
7. Verify that there is a line chart in the Total Likes and Total Comments cards.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
